lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -430,7 +430,7 @@ data SomeByronWalletPostData
 data ByronWalletPostData mw = ByronWalletPostData
     { mnemonicSentence :: !(ApiMnemonicT mw)
     , name :: !(ApiT WalletName)
-    , passphrase :: !(ApiT (Passphrase "raw"))
+    , passphrase :: !(Maybe (ApiT (Passphrase "byron-raw")))
```

Comment:
Why this change? The "standard" API shouldn't allow creating Byron wallets with an empty passphrase, this should only be possible through the legacy endpoint when providing the encrypted XPrv. Sorry for the confusion, I misread your question on slack :(, I thought you were talking about the put passphrase endpoint.

Reply:
ok, then we cannot test it easily. I mean now we can and it passes but I can remove the last two commits and should be fine

Reply:
Hmmm. Okay, let's keep it.